go/api/server.go

```diff
@@ -178,6 +193,9 @@ func setRoutes() {
 	tripPhotosRoutes.DELETE("/:photoID", middlewares.UserHasTripEditRight, middlewares.PhotoBelongsToTrip, photoHandler.DeletePhotoFromTrip)
 	tripPhotosRoutes.GET("/:photoID/download", middlewares.UserIsTripParticipant, middlewares.PhotoBelongsToTrip, photoHandler.DownloadPhoto)
 
+	featuresRoutes := router.Group("/app-settings", middlewares.UserIsLogged)
```

Comment:
```suggestion
	featuresRoutes := router.Group("/app-settings")
```

Peut etre qu'on devrait pas mettre de middleware la dessus, si par exemple on veut désactiver la connection